multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -16,15 +16,9 @@ coreos:
     - name: etcd2.service
       command: start
 
-    - name: docker.service
```

Comment:
Previously, Kubernetes used docker bridge networking for containers. This worked because flannel  obtains a subnet for the host and configures the docker bridge to use that subnet. Then docker will assign IPs from that range.

Now that Kubernetes is configured to use CNI networking, it doesn't use Docker for networking. Docker creates the containers with `--net=none` then K8s invokes the CNI plugin to do the networking. 

Now the flannel CNI plugin is used which calls through to the Calico CNI plugin to do the actual container networking. The flannel plugin tells Calico that it should call through to the host-local IPAM plugin for IP assignment, and flannels also configures the IP range that is passed to the host-local plugin.

Hence, docker no longer has a dependency on Flannel.